### PR TITLE
More item AI decomp

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**

### CODE REVIEW

File: include/dungeon_entity.h [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #define GUARD_DUNGEON_ENTITY_H
 
 #include "constants/move.h"
+#include "global.h"
```
Comment (SethBarberee) [inline include/dungeon_entity.h:5]: Why is `global.h` included here? I'd rather stick to including global in each `.c` file like `pokeemerald`

Comment (AnonymousRandomPerson) [reply include/dungeon_entity.h:5]: Oh, this should be `constants/global.h`. `MAX_MON_MOVES` is defined there and used in the entity struct.

Comment (AnonymousRandomPerson) [reply include/dungeon_entity.h]: Fixed now.

File: include/pokemon_3.h [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#ifndef GUARD_POKEMON_3_H
+#define GUARD_POKEMON_3_H
+
+#include "global.h"
```
Comment (SethBarberee) [inline include/pokemon_3.h:4]: Ditto for this header too

Comment (AnonymousRandomPerson) [reply include/pokemon_3.h]: Removed.

**Pull request merged**